contrib/babelfishpg_common/src/collation.c

```diff
@@ -1400,6 +1455,7 @@ get_collation_callbacks(void)
 		collation_callbacks_var.find_cs_as_collation_internal = &find_cs_as_collation;
 		collation_callbacks_var.find_collation_internal = &find_collation;
 		collation_callbacks_var.has_ilike_node = &has_ilike_node;
+		collation_callbacks_var.translate_collation_bbf = &translate_collation_bbf;
```

Comment:
We have to initialise it at https://github.com/amazon-aurora/babelfish_extensions/blob/jira-BABEL-3838/contrib/babelfishpg_common/src/collation.c#L27